Compilation error due to missing g++ dependency

When we try to compile the project for the first time in a newly installed virtual machine, the following error is shown:

```bash
make
```
```bash
  CC       src/libespeak-ng/la-wavegen.lo
  CC       src/libespeak-ng/la-klatt.lo
  CC       src/libespeak-ng/la-sPlayer.lo
  CXX      src/speechPlayer/src/frame.lo
./libtool: line 1917: g++: command not found
make[1]: *** [Makefile:2265: src/speechPlayer/src/frame.lo] Erro 1
make[1]: Saindo do diret\Uffffffff '/home/user/git/espeak-ng'
make: *** [Makefile:963: all] Erro 2
```

The virtual machine has a clean installation of Debian-based operating system (Ubuntu).

--- a/docs/building.md
+++ b/docs/building.md
@@ -93,7 +93,7 @@ be installed using the following commands:
 | Dependency    | Install                                                          |
 |---------------|------------------------------------------------------------------|
 | autotools     | `sudo apt-get install make autoconf automake libtool pkg-config` |
-| c99 compiler  | `sudo apt-get install gcc`                                       |
+| c99 compiler  | `sudo apt-get install gcc g++`                                   |
 | sonic         | `sudo apt-get install libsonic-dev`                              |
 | ronn          | `sudo apt-get install ronn`                                      |
 | kramdown      | `sudo apt-get install kramdown`                                  |

